### ISSUE TITLE
Problems with subscriptions

### ISSUE THREAD

**Issue by ertrzyiks** [logux/logux-server#45]
On the client side I use this:
```
client.log.add({ type: 'logux/subscribe', name: 'user/10' })
```

it fails and suggests to use `sync`, so I updated to:
```
client.log.add({ type: 'logux/subscribe', name: 'user/10' }, {sync: true})
```

and now it's rejected by the server as using unknown type [here](https://github.com/logux/logux-server/blob/775148a1742e60869f4f851581b2ae6656bb70d7/server-client.js#L254). 

I thought it may be a bug, but [there is spec](https://github.com/logux/logux-server/blob/775148a1742e60869f4f851581b2ae6656bb70d7/test/server-client.test.js#L419) treating this behavior as intended.

What am I doing wrong? Is there any example of successful subscription from client to server?

**Comment by ai** [comment]
Strange O_o.

Can I ask you (since you can repeat issue) to debug this code block: https://github.com/logux/logux-server/blob/master/base-server.js#L174-L200

If you didn’t find anything, please put `throw new Error('')` [here](https://github.com/logux/logux-server/blob/master/base-server.js#L588) and give me stack trace.

**Comment by ertrzyiks** [comment]
I put `console.log` before `if (action.type === 'logux/subscribe') {` as my very first debug attempt and it was never reached. Only then I found that it's rejected by `ServerClient` filter even before going to that code in base-server. 

I will provide you a stack trace as soon as I can.

**Comment by ertrzyiks** [comment]
```
unknownType (action, meta) {
    throw new Error(action.type)
```

```
 FATAL  logux/subscribe at 2017-08-29 09:51:17
        at KetchupServer.unknownType (/Users/mateuszderks/workspace/ketchup-server/node_modules/logux-server/base-server.js:588:11)
        at ServerClient.filter (/Users/mateuszderks/workspace/ketchup-server/node_modules/logux-server/server-client.js:254:16)
        at /Users/mateuszderks/workspace/ketchup-server/node_modules/logux-sync/messages/sync.js:61:31

```

**Comment by ai** [comment]
Thanks. I will look after few hours.

**Comment by ai** [comment]
Thanks for testing :).

I found source and fixed it fe53f5e

**Comment by ertrzyiks** [comment]
It works, thank you!
